cookbooks/swift/recipes/setup.rb

```diff
@@ -124,7 +124,6 @@
 if node['zipkin'] then
   [
     "git+https://github.com/swiftstack/py_zipkin/",
```

Comment:
I guess we could make a case this fork should also be cloned into dev

OTOH, depending on the nature of the diff we might consider if we want to outsource maintenance of this dependency - the maintainer seems to be doing releases:

https://pypi.org/project/py-zipkin/

Reply:
I think things are to the point that we can use upstream now -- but it looks like we want the `work-with-py-zipkin-upstream-0.19.0` branch of swift-zipkin now? I'll poke at it a bit.